sdks/python/apache_beam/ml/inference/pytorch.py

```diff
@@ -68,13 +71,14 @@ class PytorchModelLoader(ModelLoader):
   def __init__(
       self,
       state_dict_path: str,
-      model_class: torch.nn.Module,
+      model_class: Callable,
```

Comment:
nit: you might document that this is expected to return a `torch.nn.Module` by changing this typehint:
```suggestion
      model_class: Callable[[...],torch.nn.Module],
```